docs/ops/condition/NonZero_3.md

```diff
@@ -4,34 +4,37 @@
 
 **Category**: Condition operation
 
-**Short description**: *NonZero* returns the indices of the non-zero elements of the input tensor.
+**Short description**: *NonZero* returns the indices of the non-zero elements of the input tensor. 
 
 **Detailed description**: *NonZero* returns the indices of the non-zero elements of the input tensor (in row-major order - by dimension).
-The output tensor has shape `[rank(input), num_non_zero]`. For example, for the tensor `[[1, 0], [1, 1]]` the output will be `[[0, 1, 1], [0, 0, 1]]`.
+* The output tensor has shape `[rank(input), num_non_zero]`.
+* For example, for the tensor `[[1, 0], [1, 1]]` the output will be `[[0, 1, 1], [0, 0, 1]]`.
+* Each output column represents a single non-zero element and for that column, value in row `i` represents this element's index in input's `i`'th dimension.
```

Comment:
If the input tensor has rank 3 then there is no output "column". I would rather use a word "tuple" describing the last dimension with non-zero element indices.

Reply:
Tried to explain it further using "tuple"